tests/vmcheck/test-basic.sh

```diff
@@ -92,3 +92,21 @@ fi
 assert_file_has_content err.txt "chronologically older"
 echo "ok failed to upgrade to older commit"
 
+# https://github.com/projectatomic/rpm-ostree/issues/365
+vm_build_rpm test-conflict \
+    files /usr/app \
+    install "mkdir -p %{buildroot}/usr/app
+             touch %{buildroot}/usr/app/conflict-file"
```

Comment:
I could imagine in the future that we actually support merging files that have the same content...this comes up today with "multilib" in RPM where one can install both e.g. `glibc.x86_64` and `glibc.i686` and files which are identical are merged.

So how about `echo one >conflict-file` and `echo two > conflict-file` or so?